### PR TITLE
Add frontend docker code

### PR DESCRIPTION
**Pull request by AdrianJendo**
Running commands:
```
docker build -t lynx-frontend:dev --target=dev .
docker build -t lynx-frontend:prod --target=prod .
docker run -p 3000:3000 lynx-frontend:<tag>
```

### CODE REVIEW

**Comment by cameronkinsella** [comment]
Made some modifications so the frontend will generate static builds and the backend will serve this static site in production.

**Comment by AdrianJendo** [comment]
@ABarroso647 approve

File: backend/config/db.go
Comment (ABarroso647) [inline backend/config/db.go:1]: Im confused u seem to be redefining err here???

Comment (cameronkinsella) [reply backend/config/db.go:1]: yeah, I'm just not checking for errors on that err. However, since I'm not using it I should have made it `_`. I will fix.

**Review by ABarroso647 — APPROVED** [review]

**Pull request merged**